### ISSUE TITLE
Incorrect document page

### ISSUE THREAD

**Issue by Jiefei-Wang** [Bioconductor/SummarizedExperiment#41]
Hi,

When typing `?SummarizedExperiment` in R it brings the document of `RangedSummarizedExperiment`. I originally thought it was made on purpose, but from the description of `RangedSummarizedExperiment`,

> RangedSummarizedExperiment is a subclass of SummarizedExperiment and, as such, all the methods documented in ?SummarizedExperiment also work on a RangedSummarizedExperiment object. The methods documented below are additional methods that are specific to RangedSummarizedExperiment 

I think this is not intentional. There is a separate help page of `SummarizedExperiment ` and can be accessed via `?"SummarizedExperiment-class"`. Perhaps `?SummarizedExperiment ` can be redirected to `?"SummarizedExperiment-class"`?

Best,
Jiefei

**Comment by hpages** [comment]
Thanks Jiefei.

It's on purpose because this is where the `SummarizedExperiment()` constructor function is documented. Depending on how this constructor is called it will return either a SummarizedExperiment or a RangedSummarizedExperiment instance. Not super clean (and maybe a cleaner situation would be to have 2 separate constructors) but the reason for this design was to preserve backward compatibility when we did the big SummarizedExperiment refactoring 8 years ago (FWIW I [commented](https://github.com/Bioconductor/SummarizedExperiment/issues/29#issuecomment-614533424) recently on this refactoring).

I just pushed [a change](https://github.com/Bioconductor/SummarizedExperiment/commit/43651ddb878ada6595920208faaa4be47df33b46) where I replace `?SummarizedExperiment` with `class?SummarizedExperiment` in `man/RangedSummarizedExperiment-class.Rd` so the command actually opens the intended page.
